app/src/main/res/values/color_defs.xml

```diff
@@ -146,4 +146,12 @@
   <color name="admin_highlighted">#A5D3EC</color>
   <color name="coming_soon_end">#674172</color>
   <color name="coming_soon_start">#7659B6</color>
+  <!--DARK MODE-->
+  <color name="oppia_white_subtext">#C4C4C4</color>
+  <color name="oppia_primary_2">#3bd1c4</color>
```

Comment:
We should list the actual color here, not 'primary' (that's more like color_palette).

Reply:
Done.